.travis.yml

```diff
@@ -21,7 +21,6 @@ matrix:
     - julia: nightly
   fast_finish: true
   include:
-    - julia: 0.7
     - julia: 1.1
```

Comment:
```suggestion
     - julia: 1.0
     - julia: 1.1
```
?

Reply:
Nope, Julia 1.0 is already included in the default matrix.

Look at https://travis-ci.org/invenia/LibPQ.jl/builds/524962467 to see the resolved list of jobs

Reply:
Huh, okay